src/components/profile/userDetailsForm/userImage/userImage.tsx

```diff
@@ -39,30 +28,20 @@ export default function UserImage({ profileImg }: { profileImg?: string }) {
   return (
     <div className={styles.userImageContainer}>
       <div className={styles.imageSectionContainer}>
-        {showLoading ? (
-          <div className={styles.loader}>
-            <img src={loadingAnimation} alt="loading" />
+        {uploadedImg && <img className={styles.profileImage} src={uploadedImg} alt="profile-img" />}
+        {!uploadedImg && profileImg && <img className={styles.profileImage} src={profileImg} alt="profile-img" />}
+        {!uploadedImg && !profileImg && (
+          <div className={styles.noImageContainer}>
+            <div className={styles.noImageBox}>
+              <p className={styles.noImageText}>No Image</p>
+            </div>
           </div>
-        ) : (
-          <>
-            {uploadedImg && <img className={styles.profileImage} src={uploadedImg} alt="profile-img" />}
-            {!uploadedImg && profileImg && <img className={styles.profileImage} src={profileImg} alt="profile-img" />}
-            {!uploadedImg && !profileImg && (
-              <div className={styles.noImageContainer}>
-                <div className={styles.noImageBox}>
-                  <p className={styles.noImageText}>No Image</p>
-                </div>
-              </div>
-            )}
-          </>
         )}
       </div>
       <Form>
         <div className={styles.fileFormContainer}>
           <input className={styles.fileUploadInput} type="file" name="profileImg" ref={fileInputRef} />
-          <Button onClick={() => handleUpload()} submit={false}>
-            Change profile image
-          </Button>
+          <Button onClick={() => handleUpload()}>Change profile image</Button>
```

Comment:
Extra function here. Possible to simplify
`onClick={() => handleUpload()}`
TO
`onClick={handleUpload}`